using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -155,182 +167,198 @@ func application(_ application: UIApplication, didRegisterForRemoteNotifications
 {% endtab %}
 {% endtabs %}
 
-## trackNotificationReceive
+## trackNotificationClick
```

Comment:
The tab formatting is off in the current docs - https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference#tracknotificationclick. Verify this is fixed in your pr.

Reply:
I believe this should be fixed in this pr.